addons/im_livechat/static/src/legacy/widgets/thread.js

```diff
@@ -455,9 +452,9 @@ var ThreadWidget = Widget.extend({
             placement: 'auto',
             trigger: 'hover',
             offset: '0, 1',
-            content: function () {
-                var messageID = $(this).data('message-id');
-                var message = _.find(messages, function (message) {
+            content() {
+                const messageID = $(this).data('message-id');
+                const message = messages.find(function (message) {
```

Comment:
The inner functions that don't use `this`/`self` could have arrow function, so the code is simpler.
Here for example it would just be `messages.find(message => message.getID() === messageID)`